benchmarks/bench_plot_onnxruntime_decision_tree.py

```diff
@@ -0,0 +1,229 @@
+# coding: utf-8
+"""
+Benchmark of onnxruntime on DecisionTree.
+"""
+# Authors: Xavier Dupré (benchmark)
+# License: MIT
```

Comment:
Have we agreed on this style? Shall I adopt this in my changes as well?

Reply:
It is a proposal. scikit-learn does that.